alibi/explainers/partial_dependence.py

```diff
@@ -676,26 +689,38 @@ def plot_pd(exp: Explanation,
         A parameter specifying whether the y-axis of the PD and ICE curves should be on the same scale
         for several features. Possible values are: ``'all'`` | ``'row'`` | ``None``.
     pd_num_kw
-        Keyword arguments passed to the `matplotlib plot` function when plotting the PD for a numerical feature.
+        Keyword arguments passed to the `matplotlib.pyplot.plot` function when plotting the PD for a
+        numerical feature.
     ice_num_kw
-        Keyword arguments passed to the `matplotlib plot` function when plotting the ICE for a numerical feature.
+        Keyword arguments passed to the `matplotlib.pyplot.plot` function when plotting the ICE for a
+        numerical feature.
     pd_cat_kw
-        Keyword arguments passed to the `matplotlib plot` function when plotting the PD for a categorical feature.
+        Keyword arguments passed to the `matplotlib.pyplot.plot` function when plotting the PD for a
+        categorical feature.
     ice_cat_kw
-        Keyword arguments passed to the `matplotlib plot` function when plotting the ICE for a categorical feature.
+        Keyword arguments passed to the `matplotlib.pyplot.plot` function when plotting the ICE for a
+        categorical feature.
     pd_num_num_kw
-        Keyword arguments passed to the `matplotlib contourf` function when plotting the PD for two numerical features.
+        Keyword arguments passed to the `matplotlib.pyplot.contourf` function when plotting the PD for two
+        numerical features.
     pd_num_cat_kw
-        Keyword arguments passed to the `matplotlib plot` function when plotting the PD for a numerical and a
+        Keyword arguments passed to the `matplotlib.pyplot.plot` function when plotting the PD for a numerical and a
         categorical feature.
     pd_cat_cat_kw
-        Keyword arguments passed to the `seaborn heatmap` functon when plotting the PD for two categorical features.
+        Keyword arguments passed to the :py:meth:`alibi.utils.visualization.heatmap` functon when plotting the PD for
+        two categorical features.
     fig_kw
-        Keyword arguments passed to the `fig.set` function.
+        Keyword arguments passed to the `matplotlib.figure.set`_ function.
+
+        .. _matplotlib.pyplot.plot:
+            https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.plot.html
+
+        .. _matplotlib.figure.set:
+            https://matplotlib.org/stable/api/figure_api.html
```

Comment:
This is not showing in hyperlinks, probably because the references are missing a `_`.